ush/ioda/bufr2ioda/bufr2ioda_satwind_amv_avhrr.py

```diff
@@ -0,0 +1,471 @@
+            obsspace.write_attr('platformCommonName', satellite_name)
+            obsspace.write_attr('sensorCommonName', sensor_name)
+            obsspace.write_attr('processingLevel', process_level)
+            obsspace.write_attr('platformLongDescription', platform_description)
+            obsspace.write_attr('sensorLongDescription', sensor_description)
+
+            # Create IODA variables
+            logger.debug('Write variables: name, type, units, and attributes')
+            # Longitude
+            obsspace.create_var('MetaData/longitude', dtype=lon2.dtype, fillval=lon2.fill_value) \
+                .write_attr('units', 'degrees_east') \
+                .write_attr('valid_range', np.array([-180, 180], dtype=np.float32)) \
+                .write_attr('long_name', 'Longitude') \
+                .write_data(lon2)
+
+            # Latitude
+            obsspace.create_var('MetaData/latitude', dtype=lat.dtype, fillval=lat2.fill_value) \
+                .write_attr('units', 'degrees_north') \
+                .write_attr('valid_range', np.array([-90, 90], dtype=np.float32)) \
+                .write_attr('long_name', 'Latitude') \
+                .write_data(lat2)
+
+            # Datetime
+            obsspace.create_var('MetaData/dateTime', dtype=np.int64, fillval=int64_fill_value) \
+                .write_attr('units', 'seconds since 1970-01-01T00:00:00Z') \
+                .write_attr('long_name', 'Datetime') \
+                .write_data(timestamp2)
+
+            # Satellite Identifier
+            obsspace.create_var('MetaData/satelliteIdentifier', dtype=satid2.dtype, fillval=satid2.fill_value) \
+                .write_attr('long_name', 'Satellite Identifier') \
+                .write_data(satid2)
+
+            # Sensor Zenith Angle
+            obsspace.create_var('MetaData/satelliteZenithAngle', dtype=satzenang2.dtype, fillval=satzenang2.fill_value) \
+                .write_attr('units', 'degree') \
+                .write_attr('valid_range', np.array([0, 90], dtype=np.float32)) \
+                .write_attr('long_name', 'Satellite Zenith Angle') \
+                .write_data(satzenang2)
+
+            # Sensor Centrall Frequency
+            obsspace.create_var('MetaData/sensorCentralFrequency', dtype=chanfreq2.dtype, fillval=chanfreq2.fill_value) \
+                .write_attr('units', 'Hz') \
+                .write_attr('long_name', 'Satellite Channel Center Frequency') \
+                .write_data(chanfreq2)
+
+            # Data Provider
+            obsspace.create_var('MetaData/dataProviderOrigin', dtype=ogce2.dtype, fillval=ogce2.fill_value) \
+                .write_attr('long_name', 'Identification of Originating/Generating Center') \
+                .write_data(ogce2)
+
+            # Quality: Percent Confidence - Quality Information Without Forecast
+            obsspace.create_var('MetaData/qualityInformationWithoutForecast', dtype=qifn2.dtype, fillval=qifn2.fill_value) \
```

Comment:
According to the ObsSpace.yaml, I think this should be "MetaData/qiWithoutForecast"

https://github.com/JCSDA-internal/ioda/blob/829c25ab54f27717d65149a7be0eabaf5abcc123/share/ioda/yaml/validation/ObsSpace.yaml#L1150

Reply:
This will likewise need to be changed across-the-board for AMVs, since they all utilize `qualityInformationWithoutForecast` in converters as well as YAML filters.

@CoryMartin-NOAA @RussTreadon-NOAA - any thoughts on how to proceed here? It's a small change in this case since QI isn't used to QC these observations, but it will affect the satwinds from GEOs

Reply:
Following UFO meeting today, I will open an issue in jcsda-internal/ioda to ask about adding `qualityInformationWithoutForecast` as a valid variable name in ObsSpace.yaml

Reply:
I am going to mark this as resolved, pending action by JCSDA/ioda team on https://github.com/JCSDA-internal/ioda/issues/1233